Library/Formula/vdirsyncer.rb

```diff
@@ -0,0 +1,62 @@
+class Vdirsyncer < Formula
+  desc "Synchronize calendars and contacts"
+  homepage "https://github.com/untitaker/vdirsyncer"
+  url "https://github.com/untitaker/vdirsyncer/archive/0.5.2.tar.gz"
+  sha256 "0992d4c0c98e71a0638050f3900125a82bd29e15d5be080a4520830fa48c660a"
+
+  option "without-keyring", "Build without python-keyring support"
+  depends_on :python3
+
+  if build.with? "keyring"
```

Comment:
Don't wrap resources in the main block. They should be wrapped inside `def install` instead.